ouroboros-network/src/Ouroboros/Network/PeerSelection/RootPeersDNS.hs

```diff
@@ -24,6 +24,9 @@ module Ouroboros.Network.PeerSelection.RootPeersDNS (
     DNS.Domain,
     DNS.TTL,
     IPv4,
+
+    -- * Socket type re-exports
+    Socket.PortNumber,
```

Comment:
@coot Thanks to this extra re-export and the re-exports in `Ouroboros.Consensus.Ledger.SupportsPeerSelection`, it should not be necessary to import anything from the network layer or network-related packages in `Ouroboros.Consensus.Shelley.Ledger.PeerSelection`.